packages/skia-draw/src/premades/ImageMarkup/EDSControlPanel.tsx

```diff
@@ -139,14 +139,14 @@ export const EDSControlPanel = (props: {
                 key={"undo-button"}
                 style={styles.buttonStyle}
             >
-                <Typography color="gray">undo</Typography>
+                <Typography color="secondary">undo</Typography>
```

Comment:
Typography's color field I think should not be used with secondary/danger etc. In EDS, this field accepts HTML colors(hex values and named colors like blue, salmon, tomato etc)

Reply:
It seems Typography in EDS supports both Hex codes and shorthand codes like this (primary, secondary, danger, warning, etc..). You can verify this in the [eds storybook](https://eds-storybook-react.azurewebsites.net/?path=/docs/typography-typography--introduction#typography)
I'll add support for hex codes as well 👌

Reply:
Fixed in ca9a088